src/layouts/Unauthenticated/forms/CreateAccountForm.tsx

```diff
@@ -31,6 +32,7 @@ const CreateAccountForm: SFC = () => {
         username: values.username,
       };
       const user = await dispatch(createUser(requestData));
+      await dispatch(getIa());
```

Comment:
We can also remove `await` so fetching and storing ia can be done asynchronously.
```suggestion
      dispatch(getIa());
```

@buckyroberts What do you suggest?